taf/updater/updater.py

```diff
@@ -797,20 +802,23 @@ def _update_authentication_repository(repository_updater):
                     target_filepath,
                     current_commit,
                 )
-    except Exception as e:
-        # for now, useful for debugging
-        taf_logger.error(
-            "Validation of authentication repository {} failed at revision {} due to error {}",
-            users_auth_repo.name,
-            current_commit,
-            e,
-        )
-
-        raise UpdateFailedError(
-            f"Validation of authentication repository {users_auth_repo.name}"
-            f" failed at revision {current_commit} due to error: {e}"
-        )
+        except Exception as e:
+            metadata_expired = EXPIRED_METADATA_ERROR in type(
```

Comment:
`ExpiredMetadataError` is not raised in the newest version of the updater that we just merged in, so I don't think that this will work after merging `master`. Would it be easier to handle everything on the handler's side?

Reply:
Actually, it does raise! Not very different error handling than the old TUF.